src/main.js

```diff
@@ -23,31 +24,36 @@ var load = loader({
   IndexedTask: {
     requires: ['cfg', 'monitor'],
     setup: ({cfg, monitor}) => data.IndexedTask.setup({
-      account:          cfg.app.azureAccount,
-      table:            cfg.app.indexedTaskTableName,
-      credentials:      cfg.taskcluster.credentials,
-      authBaseUrl:      cfg.taskcluster.authBaseUrl,
-      monitor:          monitor.prefix('table.indexedtasks'),
+      tableName:    cfg.app.indexedTaskTableName,
+      credentials:  sasCredentials({
+        accountId:  cfg.azure.accountId,
+        tableName:  cfg.app.indexedTaskTableName,
+        rootUrl:    cfg.taskcluster.rootUrl,
+        credentials:cfg.taskcluster.credentials,  
+      }),
     }),
   },
+
   Namespace: {
     requires: ['cfg', 'monitor'],
     setup: ({cfg, monitor}) => data.Namespace.setup({
-      account:          cfg.app.azureAccount,
-      table:            cfg.app.namespaceTableName,
-      credentials:      cfg.taskcluster.credentials,
-      authBaseUrl:      cfg.taskcluster.authBaseUrl,
-      monitor:          monitor.prefix('table.namespaces'),
+      tableName:    cfg.app.namespaceTableName,
+      credentials:  sasCredentials({
+        accountId:  cfg.azure.accountId,
+        tableName:  cfg.app.namespaceTableName,
+        rootUrl:    cfg.taskcluster.rootUrl,
+        credentials:cfg.taskcluster.credentials,  
+      }),
     }),
   },
 
   // Create a validator
-  validator: {
+  schemaset: {
```

Comment:
https://pastebin.mozilla.org/9087682
Getting this error for `taskcluster-lib-validate` used as `schemasrt`

Reply:
It's because `schemas/constants.yml` got renamed to `schemas/v1/constants.yml`.  Move it back into `schemas/` and the error should disappear.